src/components/table/__spec__.js

```diff
@@ -885,15 +885,15 @@ describe('Table', () => {
     describe('if there are children that are immutable', () => {
       it('returns the children if there are children', () => {
         let data = Immutable.fromJS([{ foo: 1 }, { foo: 2 }]),
```

Comment:
If a variable is not intended to be reassigned it should be declared with `const`